src/main/resources/add-ons-to-index.json

```diff
@@ -2276,6 +2302,7 @@
       "uid": "org.openmrs.module.patientmatching",
       "type": "OMOD",
       "name": "Patient Matching",
+      "icon": "handshake-o",
```

Comment:
This one isn't really relevant so I guess this should be replaced/ deleted. The link icon makes more sense here. To understand why, check out the wiki of the module here https://wiki.openmrs.org/display/docs/Patient+Matching+Module